Add: Adding shared capabilities platformName and acceptInsecureCerts (#326)
SeleniumHQ/seleniumhq.github.io: docs_source_files/content/driver_idiosyncrasies/shared_capabilities.de.md

| -------- | ----------- | ----- |
| normal | complete | Used by default, waits for all resources to download |
| eager | interactive | DOM access is ready, but other resources like images may still be loading |
| none | Any | Does not block WebDriver at all |

## platformName

This identifies the operating system at the remote-end, 
fetching the `platformName` returns the OS name. 

> In could-based providers, 

Reviewer: ```suggestion
In cloud-based providers, 
```

Author: Updated in all pages!

> In could-based providers, 

Reviewer 2: cloud

Author: Thanks a lot @cgoldberg. I have updated in all translated pages!

> In could-based testing(SauceLabs or BrowserStack), 

Reviewer: ```suggestion
In cloud-based providers, 
```

Reviewer: I would not mention Sauce Labs or BrowserStack because it would feel unfair to all the other ones who are in the market... So let's just use cloud-based providers.

Author: Got it. Updated as suggested @diemol.

> setting `platformName` sets the os at the remote-end 

Reviewer: ```suggestion
setting `platformName` sets the OS at the remote-end 
```

Author: Updated in all pages!

## acceptInsecureCerts

This capability checks whether expired (or) 
invalid `TLS Certificate` are used while navigating 

> during a session.

Reviewer: ```suggestion
This capability checks whether an expired (or) 
invalid `TLS Certificate` is used while navigating 
during a session.
```

Author: Updated in all pages!

If the capability is set to `false`, an 
[insecure certificate error](//developer.mozilla.org/de/docs/Web/WebDriver/Errors/InsecureCertificate) 
will be returned as navigation encounters any domain 

> certificate problems. If set to `ture`, invalid certificate will be 

Reviewer: ```suggestion
certificate problems. If set to `true`, invalid certificate will be 
```

Author: Updated in all pages!

trusted by the browser.

All self-signed certificates will be trusted by this capability by default. 

> Onc set, `acceptInsecureCerts` capability will have effect for entire session

Reviewer: ```suggestion
Once set, `acceptInsecureCerts` capability will have an effect for the entire session.
```

Author: Updated in all pages!
